kubernetes/resource_kubernetes_pod_test.go

```diff
@@ -109,6 +109,58 @@ func TestAccKubernetesPod_updateForceNew(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesPod_updateEnvForceNew(t *testing.T) {
+	var conf api.Pod
+
+	podName := fmt.Sprintf("tf-acc-test-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
```

Comment:
I think you could use RandString here, I don't think you need to specify the charset, but I could be wrong.